src/ranch.erl

```diff
@@ -42,6 +42,7 @@
 -export([info/0]).
 -export([info/1]).
 -export([procs/2]).
+-export([stats/1]).
```

Comment:
I am not convinced this needs to be exported.

Reply:
And ditto.

Reply:
and done